### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by PhilPJL** [BottlecapDave/HomeAssistant-OctopusEnergy#1083] [General]
@BottlecapDave  I tried to sponsor you yesterday I think it was for $20 one off.  I didn't get a notification but today I've got a credit card transaction of 'Paypal *Dave' for £52.  Did you receive anything from me?  There's no record on GitHub or PayPal.

**Comment by BottlecapDave** [comment]
Hello thanks for the attempt :) 

You're not appearing on the sponsorship dashboard. $20 also shouldn't result in a £52 charge. It also appears GitHub hasn't supported PayPal for a while - https://github.blog/changelog/2023-01-23-github-sponsors-will-stop-supporting-paypal/

**Reply by PhilPJL** [reply]
OK panic over. My wife bought something for £52. No idea why it showed up as *Dave. Unlucky coincidence :) I'll try again with the sponsorship.
